src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headerRow = table.tHead.children[0];
+const people = [...table.tBodies[0].querySelectorAll('tr')];
+
+headerRow.addEventListener('click', handler);
+
+function handler(event) {
+  const header = event.target;
+  const headerValues = ['Name', 'Position', 'Age', 'Salary'];
+
+  const sortIndex = headerValues.indexOf(header.innerText);
```

Comment:
each table column has its own cellIndex prop (event.target.cellIndex), you can use it instead of creating array and using indexOf method on it